front/src/modules/ui/board/components/BoardColumn.tsx

```diff
@@ -21,6 +21,7 @@ export const StyledColumn = styled.div<{ isFirstColumn: boolean }>`
   min-width: 200px;
 
   padding: ${({ theme }) => theme.spacing(2)};
+  position: relative;
```

Comment:
The position of the `BoardColumnMenu`, which is `absolute` positioned, gets affected when certain columns in the KanBan board are hidden. To address this problem, setting the `BoardColumn` as the closest positioned ancestor by giving it a `position: relative` style resolves the issue.